src/operator/pad-inl.h

```diff
@@ -37,21 +37,18 @@ struct PadParam : public dmlc::Parameter<PadParam> {
         .add_enum("constant", pad_enum::kConstant)
         .add_enum("edge", pad_enum::kEdge)
         .describe(
-            "Padding type to use. \"constant\" pads all values with a constant "
-            "value, the value of which can be specified with the "
-            "constant_value option. \"edge\" uses the boundary values of the "
-            "array as padding.");
+            "Padding type to use."
+            " \"constant\" pads with `constant_value` and"
+            " \"edge\" pads using the edge values of the input array.");
 
     DMLC_DECLARE_FIELD(pad_width).describe(
-        "A tuple of padding widths of length 2*r, where r is the rank of the "
-        "input tensor, specifying number of values padded to the edges of each "
-        "axis. (before_1, after_1, ... , before_N, after_N) unique pad widths "
-        "for each axis. Equivalent to pad_width in numpy.pad, but flattened.");
+        "Number of values padded to the edges of each axis. "
```

Comment:
I believe this is not factually correct. The padding width is not the "number of values". For exmple if you padded a matrix 10 by 10 with (3, 3, 3, 3) That would correspond to 156 *values* (in the sense of the number of components in the padded region. Better to say:

"Widths of the padding regions applied to the edges of each axis"